ui/pages/onboarding-flow/create-password/__snapshots__/create-password.test.js.snap

```diff
@@ -38,6 +38,7 @@ exports[`Onboarding Create Password Render should match snapshot 1`] = `
       class="box box--margin-top-1 box--margin-bottom-1 box--flex-direction-row typography typography--h4 typography--weight-normal typography--style-normal typography--align-center typography--color-text-default"
     >
       This password will unlock your MetaMask wallet only on this device. MetaMask can not recover this password.
+      This password will unlock your MetaMask Institutional extension only.
```

Comment:
What about normal MM build?

Reply:
It's the in line above L40 the MM text, and ours in L41. Because the code fences are ignored, it renders both in the snapshot.